varcode/frameshift_coding_effect.py

```diff
@@ -118,8 +118,84 @@ def _frameshift(
         transcript=transcript,
         aa_mutation_start_offset=mutation_start_position,
         aa_ref=aa_ref,
-        shifted_sequence=mutant_protein_suffix)
+        shifted_sequence=str(mutant_protein_suffix))
 
+def cdna_codon_sequence_after_insertion_frameshift(
```

Comment:
Found the `after` in the name confusing; initially expected that I'd get the sequence literally after the insertion (given all the `after`s inside the function that mean that) vs. the sequence following application of the insertion. Feel free to ignore.